src/SchemaForm.vue

```diff
@@ -65,7 +69,7 @@ export default {
     binds (field) {
       return field.schema
         ? { schema: field.schema }
-        : field
+        : { ...field, ...this.sharedConfig }
```

Comment:
Don’t we want to reverse the order though? So that a field can override the `sharedConfig` if required.

Reply:
Good call :)